src/cladetime/tree.py

```diff
@@ -35,9 +35,19 @@ def __init__(self, clade_time: CladeTime):
         """Tree constructor."""
         self._clade_time = clade_time
         self.as_of = self._clade_time.tree_as_of
-        self._nextclade_data_url = self._clade_time._config.nextclade_data_url
-        self._nextclade_data_url_version = self._clade_time._config.nextclade_data_url_version
-        self._tree_name = self._clade_time._config.nextclade_input_tree_name
+        self._config = self._clade_time._config
```

Comment:
Note that future commits remove `CladeTime` from `Tree` instantiation to avoid circular dependencies, so all of the awkward references to `_clade_time_config` and the like will disappear